pkg/tfbridge/info.go

```diff
@@ -394,6 +394,10 @@ type SchemaInfo struct {
 
 	// whether or not to treat this property as secret
 	Secret *bool
+
+	// whether to treat this as a computed input for the purpose of diff, i.e.: should trigger an
+	// update even if it appears only as an output property.
```

Comment:
I don't know what this means outside of the context of a `Provider`. If this is provider only, let's say and enforce that. If not, can you flesh out this explanation.

Reply:
Can we mark it as experimental for now?

Reply:
Yeah, it should be experimental. Should I rename it to `XComputedInput`?